addon/mixins/legend.js

```diff
@@ -64,6 +67,15 @@ export default Ember.Mixin.create({
     return this.get('legendItemWidth') - this.get('legendIconRadius') - this.get('legendLabelPadding') * 2;
   }),
 
+  // we use legendRowWidths in order to estimate how much to move the
```

Comment:
Personal thing here, but I don't like using any kind of pronouns in documentation. I would write this as `legendRowWidths is used to estimate how much...`.